examples/hitl/rearrange_v2/habitat_llm_loader.py

```diff
@@ -38,9 +43,21 @@ def load_collaboration_episode_data(
 
         return episode_data
 
+    def register_habitat_llm_extensions(config):
+        register_actions(config)
+
+        if hasattr(config.habitat.dataset, "metadata"):
```

Comment:
So habitat-llm `register_measures` requires `config.habitat.dataset.metadata`? That isn't super-intuitive since measures don't sound related to datasets. Maybe this code would break someday if that linkage ever changes.

Reply:
This is expected to be refactored as the features are brought into `habitat-lab`.

See https://github.com/facebookresearch/habitat-lab/pull/2022.